### DISCUSSION TITLE
Permissions on macOS for ISC DHCP Configuration File

### DISCUSSION THREAD

**Discussion by Kyosho** [Homebrew/discussions#2338] [Everyday usage]
Hi - On macOS Big Sur, in order to copy in my own dhcpd.conf file to use with Homebrew's installation of ISC DHCP server, it looks like I need to copy into the directory: /opt/homebrew/Cellar/isc-dhcp/4.4.2-P1/etc/ and then symlink that config file to /opt/homebrew/etc/

I usually have to assign ownership to the **named** user and group on Linux implementations. Here, the sample config files on a fresh install are owned by admin, the administrator who installed the package.

Do I just copy this style and make admin:admin the owner:group for dhcpd.conf?

Thanks - Frank

**Comment by carlocab** [comment]
I'm unsure about your question re permissions, but you shouldn't need to install anything into `/opt/homebrew/Cellar/isc-dhcp/4.4.2-P1/etc/`. It should be sufficient to put them in `/opt/homebrew/etc`. If that doesn't work, then there's a bug in the formula.

**Comment by Kyosho** [comment]
Hi Carlo - Thank you so much for getting back to me. So there is a sample
dhcpd.conf file located in /opt/homebrew/Cellar/isc-dhcp/4.4.2-P1/etc/.

That file is symlinked into /opt/homebrew/etc.

I assume I need to create an actual DHCP configuration filed named:
*dhcpd.conf*

Am I correct so far?

The location of that file will be where? Doesn't it have to go in
/opt/homebrew/Cellar/isc-dhcp/4.4.2-P1/etc/ with a symlink to
/opt/homebrew/etc?

If yes, I want to set the correct permissions on the file I create or copy.

If this is all clear, that's my question.

Thanks - Frank


On Sat, Oct 23, 2021 at 10:47 AM Carlo Cabrera ***@***.***>
wrote:

> I'm unsure about your question re permissions, but you shouldn't need to
> install anything into /opt/homebrew/Cellar/isc-dhcp/4.4.2-P1/etc/. It
> should be sufficient to put them in /opt/homebrew/etc. If that doesn't
> work, then there's a bug in the formula.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Homebrew/discussions/discussions/2338#discussioncomment-1525262>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAZM3U33ABLGXEOA7MQH6GDUILDIDANCNFSM5GSK424A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Frank Kyosho Fallon
My pronouns are: He, HIm

**Reply by SMillerDev** [reply]
No, it has to go directly into `/opt/homebrew/etc`

**Comment by Kyosho** [comment]
Thanks! These instructions worked just right. I now have a working DHCP
failover pair.

On Sun, Oct 24, 2021 at 10:46 AM Sean Molenaar ***@***.***>
wrote:

> No, it has to go directly into /opt/homebrew/etc
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Homebrew/discussions/discussions/2338#discussioncomment-1527817>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAZM3U5V6UVQNZEPDIHCBMDUIQL4BANCNFSM5GSK424A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Frank Kyosho Fallon
My pronouns are: He, HIm
